website/src/controllers/api-v3/groups.js

```diff
@@ -34,6 +34,7 @@ api.createGroup = {
 
     let group = new Group(Group.sanitize(req.body)); // TODO validate empty req.body
     group.leader = user._id;
+    group.memberCount = 1;
```

Comment:
I have no strong opinion about it, the place where you add the first user and the leader seemed the most appropriate.